xt/01-make-update-deps.t

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env perl
-# Copyright 2022 SUSE LLC
+# Copyright SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
 
+use strict;
+use warnings 'all';
```

Comment:
Not needed because of Test::Most. And the `'all'` is also not needed. Don't get used to it. `use warnings` is just enough.